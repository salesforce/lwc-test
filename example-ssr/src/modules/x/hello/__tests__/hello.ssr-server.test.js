import Greeting from '../hello';
import { renderAndHashComponent } from '@lwc/jest-ssr-snapshot-utils';

describe('<x-hello>', () => {
    test('should render on the server', async () => {
        const { renderedComponent, snapshotHash } = renderAndHashComponent('x-hello', Greeting);
        expect(renderedComponent).toMatchSnapshot(snapshotHash);
    });
});
