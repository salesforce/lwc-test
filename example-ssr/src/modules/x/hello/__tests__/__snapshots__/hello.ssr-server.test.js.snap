// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<x-hello> should render on the server: 0270b7e859a9b2246c8fea754dc918f9cd9a2caba62f8fe68fde895056b38633 1`] = `"<x-hello><template shadowrootmode="open"><style type="text/css">div {margin: 10px;}button {margin-top: 10px;padding: 5px 10px;}</style><div><h1>Greeting Form</h1><label for="name">Your Name:</label><input id="name" type="text"><button>Submit</button></div></template></x-hello>"`;
